src/components/resumption/src/last_state_impl.cc

```diff
@@ -42,34 +42,39 @@ LastStateImpl::LastStateImpl(const std::string& app_storage_folder,
                              const std::string& app_info_storage)
     : app_storage_folder_(app_storage_folder)
     , app_info_storage_(app_info_storage) {
-  LoadStateFromFileSystem();
+  LoadFromFileSystem();
   LOG4CXX_AUTO_TRACE(logger_);
 }
 
 LastStateImpl::~LastStateImpl() {
   LOG4CXX_AUTO_TRACE(logger_);
-  SaveStateToFileSystem();
+  SaveToFileSystem();
 }
 
-void LastStateImpl::SaveStateToFileSystem() {
```

Comment:
deprecated method without definition

Reply:
@jacobkeeler Done in 4fdae6f97e8445b4aebe5269c4163e028115f508

Reply:
This comment was not resolved

Reply:
@jacobkeeler The method is marked as `DEPRECATED` and the definition has been added. Please, take a look at the 56361c87e7def67b2780721df021e4c7b98999ca